addons/web_tour/static/src/js/tip.js

```diff
@@ -316,8 +316,25 @@ var Tip = Widget.extend({
                 of: this.$anchor,
                 collision: "none",
                 using: props => {
-                    this.el.style.setProperty('top', `${props.top}px`, 'important');
-                    this.el.style.setProperty('left', `${props.left}px`, 'important');
+                    const {top} = props;
+                    let {left} = props;
+                    const anchor = this.$anchor[0];
```

Comment:
Name the variable `anchorEl` (suffix by El when it's a variable containing an element)